### PR TITLE
Corrected Migrator Image Version

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Corrected migrator image version

**Related issue(s)**

### CODE REVIEW

File: resources/keb/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ global:
       path: europe-docker.pkg.dev/kyma-project/prod
     schema_migrator:
       dir: "control-plane"
-      version: "v20231211-c10b7b03"
+      version: "v20230120-865831c1" # do not update along with the other images
```
Comment (jaroslaw-pieszka) [inline resources/keb/values.yaml:11]: Why such an old version (January!)

Comment (ralikio) [reply resources/keb/values.yaml:11]: This is just the newest version used in Control Plane.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**